d2bs/kolbot/libs/bots/Duriel.js

```diff
@@ -4,7 +4,7 @@
 *	@desc		kill Duriel
 */
 
-function Duriel() {
+function Duriel () {
```

Comment:
run jslint or eslint settings

Reply:
i'm using eslint, this rule should be removed then `space-before-function-paren": ["error", "always"]`

Reply:
my bad, actually the space is correct